hazi.html

```diff
@@ -226,6 +286,26 @@ <h3 class="set_head">Különböző méretű hangszerek</h3>
             <footer class="set_footer_header">
                 <a href="#ugras_fel" class="set_link"> Ugrás az oldal tetejére </a>
             </footer>
-        </div>    
+        </div>  
+        <script>
+            function darkMode() {
```

Comment:
Javascriptet még nem vettünk :)

Reply:
igen tudom de így tudtam megoldni :D

Reply:
Nézelődj még a media queryknél :)

Reply:
Igen eddig nem jutottam el :D egyszerűbb volt javascriptben :D

Reply:
Nekem ki van kapcsolva a javascript :( :P

Reply:
Köszi megoldottam :) Illetve a mai óra anyagát még belefűztem, mert már annó is ilyemi desing volt a fejemben, csak eszközöm nem volt :D De most a FLEX mindent megoldott (akár csak falun a szomszéd kiidegelését :D "Amikor kora reggel ébredek a flexre..."